src/components/LogPane/index.tsx

```diff
@@ -70,7 +70,7 @@ const LogPane: React.FC<LogPaneProps> = ({
           }}
           deferredMeasurementCache={cache}
           height={height}
-          overscanRowCount={200}
+          overscanRowCount={50}
```

Comment:
One thing that I might also recommend would be to change the `defaultHeight` in `RowRenderer` to 18. It's currently set to 16, but 18 is the true height of an unwrapped row. This will only help when wrap is off though! 😔